### ISSUE TITLE
default_text_search_config set at the DB level

### ISSUE THREAD

**Issue by superhac** [PostgREST/postgrest#1707]
* PostgreSQL version: (if using docker, specify the image)
   
Docker, Lastest

* PostgREST version: (if using docker, specify the image)

Rocker, Lastest

* Operating system:

Ubuntu, 20.4

Hi,

It appears that postgREST doesn't abide to the "default_text_search_config" variable when set at the DB level.  Instead it looks like it sets itself to "english" when no language is specified in a query. e.g. fts.SEARCHWORD.     Currently my DB is set with, "ALTER DATABASE mydb SET default_text_search_config = 'simple';".  So to get it work right with my ts_vector I have to specify the language via fts(simple).SEARCHWORD. 

Thanks!

**Comment by wolfgangwalther** [comment]
I looked through the code and the postgrest docs a bit - there is no reason to assume, that this should not work.

Can you post a self-contained example with all the database objects, configuration, etc. that I need to run to recreate the issue?

**Comment by superhac** [comment]
Thanks for response!  I went back after your comment and noticed that I was working in a cloned database that didn't have the default_text_config set to "simple".  Once I made that adjustment in the clone it worked like it should have.  So postgrest is abiding by that environment variable.  Sorry for the false alarm.  BTW, Great work on this project.  I've almost converted my entire API over to postgrest and I'm still in shock how much binding code I've been able to remove!
